kasa/tests/smart/modules/test_lighttransition.py

```diff
@@ -0,0 +1,86 @@
+from pytest_mock import MockerFixture
+
+from kasa import Feature, Module
+from kasa.smart import SmartDevice
+from kasa.tests.device_fixtures import get_parent_and_child_modules, parametrize
+from kasa.tests.fixtureinfo import ComponentFilter
+
+light_transition_v1 = parametrize(
+    "has light transition",
+    component_filter=ComponentFilter(
+        component_name="on_off_gradually", version=1, include_version=True
```

Comment:
Perhaps `minimum_version` and `maximum_version` (where missing max indicates x+) would be cleaner?

Reply:
Done